src/_data/translate.yml

```diff
@@ -54,106 +54,106 @@ en:
     shiftMean: Shift mean
     std: Standard Deviation
     textAreaPlaceHolder: Please enter data or drop columns from cvs file here (no
-      column titles - see sample data on dropdown)
+      column titles - see sample data in drop down menu)
     times: times
     title: One Mean
     totalNoOfSamples: 'Total # of samples:'
-    totalSamples: 'Total samples:'
+    totalSamples: 'Total # samples'
     twoTail: Two Tails
     value: Value
   oneProportion:
     Samples: Samples
     addSamples: Add samples
-    binomial: Binomial
+    binomial: Binomial Prediction
     drawSamples: Draw Samples
     maxHead: 'Max # of head:'
     meanNoOfHeads: 'Mean # of heads:'
     minHead: 'Min # of heads:'
-    noOfHeads: '# of heads in '
+    noOfHeads: '# of heads in'
     noOfSamples: '# of samples'
     probHeads: 'Probability of Heads p:'
     proportion: 'Proportion:'
     remark1: '1. Double click the graph to zoom in/out if # of tosses >= 50.'
     remark2: 2. Click the legend to hide/show corresponding info in graph.
-    remarkTitle: 'Remarks:        '
+    remarkTitle: 'Remarks:'
     reset: Reset
-    samplesInterval: '# of samples in interval: '
-    selectInterval: 'Select interval for # of heads in'
+    samplesInterval: '# of samples in interval:'
+    selectInterval: 'Select interval for # of heads in:'
     selected: Selected
-    standardDeviation: Standard Deviation
+    standardDeviation: 'Standard Deviation:'
     title: One Proportion
     tosses1: 'Tosses:'
-    tosses2: tosses
+    tosses2: 'tosses:'
     totalSamples: 'Total samples:'
   site:
-    title: 'Statistical Analysis Vizualizions for Introtuctory Statistics '
+    title: Statistical Analysis Vizualizions for Introtuctory Statistics

```

Comment:
```suggestion
    title: Statistical Analysis Visualizations for Introductory Statistics
```

Reply:
thanks!
but i will change it in the CSV and and change the yml thru script.